pkgs/by-name/rz/rzls/update.sh

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -I nixpkgs=./. --pure -i bash -p curl cacert gnused jq common-updater-scripts dotnet-sdk_8 exiftool nix
+# shellcheck shell=bash
+
+set -euo pipefail
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+old_rzls_version="$(sed -nE 's/\s*version = "(.*)".*/\1/p' ./package.nix)"
+apiUrl="https://feeds.dev.azure.com/azure-public/vside/_apis/packaging/Feeds/msft_consumption/packages/577084ea-e5be-4fad-951b-00d0b05fb170?api-version=7.2-preview.1"
+new_rzls_version="$(curl -s $apiUrl | jq -r '.versions[0].version')"
+
+echo $old_rzls_version
+echo $new_rzls_version
+
+if [[ "$new_rzls_version" == "$old_rzls_version" ]]; then
+    echo "Already up to date!"
+    exit 0
+fi
+
+buildDir=$(mktemp -d)
+#dotnet nuget add source -n rzls "https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption/nuget/v3/index.json"
```

Comment:
It's not clear why this dead code is here.  Maybe add a TODO if it's here intentionally?